src/test/java/org/hiero/sketch/TableTest.java

```diff
@@ -0,0 +1,59 @@
+package org.hiero.sketch;
+
+import org.hiero.sketch.table.*;
+import org.hiero.sketch.table.api.IColumn;
+import org.junit.Test;
+
+import static org.hiero.sketch.DoubleArrayTest.generateDoubleArray;
+import static org.hiero.sketch.IntArrayTest.generateIntArray;
+
+public class TableTest {
+
+    @Test
+    public void ColumnCompressTest() {
```

Comment:
The style guide suggests method names starting with lowercase. Class names start with uppercase.